interfaces/system_key_test.go

```diff
@@ -412,3 +412,21 @@ func (s *systemKeySuite) TestSystemKeysUnmarshalSame(c *C) {
 	c.Assert(err, IsNil)
 	c.Check(ok, Equals, true, Commentf("key1:%#v key2:%#v", key1, key2))
 }
+
+func (s *systemKeySuite) TestRemoveSystemKey(c *C) {
+	systemKeyJSON := `{}`
+
+	// write the mocked system key to disk
+	restore := interfaces.MockSystemKey(systemKeyJSON)
+	defer restore()
+	err := interfaces.WriteSystemKey()
+	c.Assert(err, IsNil)
+
+	c.Check(dirs.SnapSystemKeyFile, testutil.FilePresent)
+
+	// remove the system key
```

Comment:
Maybe a case that ensures that no error is returned if the key does not exist?

Reply:
Good idea - added in a01a8aa061

Reply:
Thank you!